frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -345,7 +350,7 @@ pub mod pallet {
 
                 consumed_weight + T::DbWeight::get().reads_writes(5, 3)
             } else {
-                T::DbWeight::get().reads(4)
+                T::DbWeight::get().reads(3)
```

Comment:
You should keep it at 4:
* PalletDisabled::get
* Self::force_era
* Self::current_era
* Self::next_era_starting_block

Reply:
Ah, I missed force_era() !